hack/cluster-openshift.sh

```diff
@@ -264,6 +281,20 @@ else
   DOCKER_SUDO=sudo
 fi
 
+# If latest Kiali release is to be installed, figure out which version that is
+if [ "${KIALI_ENABLED}" == "true" -a "${KIALI_VERSION}" == "lastrelease" ]; then
+  get_downloader
+  eval ${DOWNLOADER} /tmp/kiali-release-latest.json https://api.github.com/repos/kiali/kiali/releases/latest
+  KIALI_VERSION=$(cat /tmp/kiali-release-latest.json |\
+    grep  "tag_name" | \
+    sed -e 's/.*://' -e 's/ *"//' -e 's/",//')
+  if [ "${KIALI_VERSION}" == "" ]; then
+    echo "ERROR: Cannot determine the latest Kiali version to install"
+    exit 1
+  fi
```

Comment:
What do you think about to show how set the Kiali version to install like the step of install curl or get ?

Reply:
I added a blurb to this error message to tell the user what env var to set.